backend/models.py

```diff
@@ -44,9 +46,7 @@ def place_mines(self):
             self.compute_neighbors(r, c)
```

Comment:
i think this should go inside MineCell functionality, where you can just call MineCell.incrementNeighborMineCount

Reply:
so instead the code would look like:
```suggestion
            for r, c in mine_positions:
              self.grid[r][c] = new MineCell(lalala)
              mine.incrementNeighborMineCount
```

Reply:
I think a potential issue is that minecells don't have a view of the entire board, so incrementingneighbormine count using r, c indices doesn't work unless I can access the Board's self.grid[][] or something similar